main.js

```diff
@@ -30,11 +30,3 @@ export {
     reserve_quoted_keys,
 } from "./lib/propmangle.js";
 import "./lib/mozilla-ast.js";
-
-// TESTS
-export * from "./lib/ast.js";
-export {
-    JS_Parse_Error,
-    tokenizer,
-} from "./lib/parse.js";
-export { to_ascii } from "./lib/minify.js";
```

Comment:
I think only `to_ascii` deserves to die. JS_Parse_Error can be used to tell what exception is being thrown by terser, inside a catch statement.

Reply:
I guess Terser can use all of this inside and this imports are not required for anything rather then tests

Reply:
We can keep some of them if someone wants to use this in the CLI for example